src/Analyzer/Pass/Expression/FinalStaticUsage.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PHPSA\Analyzer\Pass\Expression;
+
+use PhpParser\Node\Expr;
+use PHPSA\Analyzer\Pass\AnalyzerPassInterface;
+use PHPSA\Context;
+use PHPSA\Definition\ClassDefinition;
+
+class FinalStaticUsage implements AnalyzerPassInterface
+{
+    /**
+     * @param Expr\StaticCall $expr
+     * @param Context $context
+     * @return bool
+     */
+    public function pass(Expr\StaticCall $expr, Context $context)
+    {
+        $classObject = $context->scope->getPointer()->getObject();
```

Comment:
`$context->scopePointer->getObject();`
The context has already a scopePointer. I'll have to check what scope is for or if we can get rid of it